src/Quahog/Client.php

```diff
@@ -325,31 +325,30 @@ private function _closeConnection()
      *
      * @param string $response
      *
-     * @return array
+     * @return Result
      */
     private function _parseResponse($response)
     {
         $splitResponse = explode(': ', $response);
 
-        $idReturn = [];
+        $id = null;
         if (!$this->_inSession) {
             $filename = $splitResponse[0];
             $message = $splitResponse[1];
-        }
-        else {
-            $idReturn = ['id' => $splitResponse[0]];
+        } else {
+            $id = $splitResponse[0];
```

Comment:
It is possible to use `list()` here or the similar array syntax, but that can be done in another PR.

Reply:
No probs - feel free to open that PR once this is merged.